text/dotnet-core/rfcs/0004-extend-dotnet-sdk.md

```diff
@@ -0,0 +1,102 @@
+# Extend .Net SDK Dependency
+
+## Proposal
+
+Reconfigure the .Net SDK dependency to be identical to the one that can be downloaded from Microsoft which includes the runtimes that the SDK is dependent on to run.
```

Comment:
What are the barriers to directly using the .Net SDK distributed by Microsoft?

Reply:
thanks for clarifying that we are trying to use the version distributed by Microsoft